src/styles/main.scss

```diff
@@ -1,3 +1,220 @@
+
 body {
+  font-family: "Roboto", sans-serif;
+
+}
+
+.header {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  
+}
+
+.container {
+  display: flex;
+}
+
+.nav {
+  display: flex;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  align-self: center;
+  margin: 10px;
+}
+
+.box {
+  display: flex;
+  margin: 10px;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+}
+
+.nav_link {
+  text-decoration: none;
+
+  &:visited {
+    color: black;
+  }
+  
+  &:hover {
+    color: #00acdc;
+    cursor: pointer;
+  }
+}
+
+.is-active {
+  color: #00acdc;
+  text-decoration: none;
+
+  &::after {
+    content: "";
+    display: block;
+    height: 4px;
+    border-radius: 8px;
+    background-color: #00acdc;
+    position: absolute;
+    width: 37px;
+    bottom: 0;
+  }
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 950px;
```

Comment:
```suggestion
  max-width: 950px;
```

Reply:
it should be calculated with calc (number of cards in a row multiplied by card width + margin)